experimental/dds/tree2/src/simple-tree/proxies.ts

```diff
@@ -874,13 +874,19 @@ interface ExtractedFactoryContent<T extends InsertableTypedNode<TreeNodeSchema>>
 export function extractFactoryContent<
 	T extends InsertableTypedNode<TreeNodeSchema> | Unhydrated<TreeNodeSchema>,
 >(content: T): ExtractedFactoryContent<T> {
-	if (isFluidHandle(content)) {
-		return { content, hydrateProxies: noopHydrator };
-	} else if (isReadonlyArray(content)) {
+	if (isReadonlyArray(content)) {
 		return extractContentArray(content) as ExtractedFactoryContent<T>;
 	} else if (content instanceof Map) {
 		return extractContentMap(content);
 	} else if (content !== null && typeof content === "object") {
+		const flexNode = tryGetEditNode(content);
+		// Only run isFluidHandle if content can't be an unhydrated node,
```

Comment:
I read the function description and code and can't quite tie this comment to what's happening here.  Must be some implicit connection between if it's a Map and being "unhydrated"?  Is "flex" a well-defined term?  How does it relate to hydration etc?